src/gameplay/glicko2.ts

```diff
@@ -0,0 +1,136 @@
+// @ts-nocheck
```

Comment:
Let's remove this so you are forced to do it well from the start! You'll find that as you learn it, it'll be easier and easier to use :) 

Typescript is a lifesaver.